testing/data_validation/get-expectation-values-for-qubit-operator.py

```diff
@@ -20,12 +20,19 @@
             == "zapata-v1-expectation_values"
         )
 
+        identity_term = workflowresult[key]["expectation-values"]["expectation_values"][
+            "real"
+        ][0]
+        assert isinstance(float(identity_term), float)
+        assert identity_term > 1.5
```

Comment:
I can't see how this test actually works. No idea why the values are between 1.5 and 2.5. It looks like it loads data from some file, but it's done with command line, so I don't even know what file.

Could you add some comments to explain what file it's loading and make it clearer?

Reply:
The data is loaded from the input file through the command line. So it gets run with `python3 data_validation_file.py workflowresult_file.json`. 

I would rather wait to update this and explain it better as a story later on since it's basically the entirety of the acceptance tests which we may redesign.

Reply:
Sounds good to me.